lib/Config.js

```diff
@@ -81,7 +81,10 @@ class Config extends EventEmitter {
         } else {
             this.bootstrapList = [];
         }
-
+        const shouldRestrictToSite = process.env.SITE_NAME;
```

Comment:
maybe `BOOTSTRAP_SITE_NAME` would be more explicit, conveying that this does indeed affect the bootstrap list...